Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -355,9 +355,9 @@ module.exports = {
       // extension
       .addBehavior(
         'Physics2Behavior',
-        t('Physics Engine 2.0 (beta)'),
-        'Physics2Behavior',
-        t('Simulate physics, the successor of the old physics behavior'),
+        t('Physics Engine 2.0'),
+        'Physics2',
```

Comment:
@Lizard-13 FYI, I've removed "Behavior" from the default name of the behavior. As it's used to identify the behavior when writing expressions (`MyObject.Physics2::Expression()`), the convention is to just have the name of what it is - the behavior is implicit here :)

Reply:
Ok, I'm confused, now the expressions are written as `MyObject.Physics2::Expression()`?, shouldn't it break the examples? they're still working and I can't use Physics2:: so far :thinking:

Reply:
This is only the default name that is applied to a behavior that was changed.
In theory (even if it's rarely used), one object can have multiple behavior of the same type applied to it. 
This won't break any example because the name is part of the object (it's a map: name -> behavior), so changing the default name only change the name of the behavior when it's added.

Reply:
Aaaah, now I get it, totally forgot that you can rename behaviors :P (I guess never needed to do it :sweat_smile: )